### ISSUE TITLE
Free Memory is reporting as 0 on Windows

### ISSUE THREAD

**Issue by getvivekv** [shirou/gopsutil#841]
On Windows system (tested on Windows 10 Pro with administrator access) the free memory is reporting as zero. This works on Linux/Ubuntu 

**To Reproduce**
```go
package main

import (
    "fmt"

    "github.com/shirou/gopsutil/mem"
)

func main() {
    v, _ := mem.VirtualMemory()

    // almost every return value is a struct
    fmt.Printf("Total: %v, Free:%v, UsedPercent:%f%%\n", v.Total, v.Free, v.UsedPercent)

    // convert to JSON. String() is also implemented
    fmt.Println(v)
}
```

On windows, ran using 'go run app.go' 

Output

```
Total: 34050084864, Free:0, UsedPercent:40.000000%
```

**Comment by Lomanic** [comment]
Thanks for this great bug report, having a ready-to-copy-paste reproducible example and sufficient information is getting quite rare these days.

Please see documentation of the type at https://godoc.org/github.com/shirou/gopsutil/mem#VirtualMemoryStat

```go
    // This is the kernel's notion of free memory; RAM chips whose bits nobody
    // cares about the value of right now. For a human consumable number,
    // Available is what you really want.
    Free uint64 `json:"free"`
```

The mem package is mostly modeled against Linux, so as I understand this, the Free field is not set on Windows as this notion doesn't apply on Windows.

Nevertheless, we are doing something different compared to psutil and I would like @shirou advice on this.

To compare, on Windows 10:

```
go run mem_issue841.go
Total: 2147012608, Free:0, UsedPercent:52.000000%
{"total":2147012608,"available":1017741312,"used":1129271296,"usedPercent":52,"free":0,"active":0,"inactive":0,"wired":0,"laundry":0,"buffers":0,"cached":0,"writeback":0,"dirty":0,"writebacktmp":0,"shared":0,"slab":0,"sreclaimable":0,"sunreclaim":0,"pagetables":0,"swapcached":0,"commitlimit":0,"committedas":0,"hightotal":0,"highfree":0,"lowtotal":0,"lowfree":0,"swaptotal":0,"swapfree":0,"mapped":0,"vmalloctotal":0,"vmallocused":0,"vmallocchunk":0,"hugepagestotal":0,"hugepagesfree":0,"hugepagesize":0}
```

```
python
Python 3.8.1 (tags/v3.8.1:1b293b6, Dec 18 2019, 23:11:46) [MSC v.1916 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import psutil
>>> psutil.virtual_memory()
svmem(total=2147012608, available=1013882880, percent=52.8, used=1133129728, free=1013882880)
```

And [psutil documentation](https://psutil.readthedocs.io/en/latest/#psutil.virtual_memory) says

> On Windows available and free are the same.

Why doesn't gopsutil copy psutil here?

**Comment by Lomanic** [comment]
We will align on psutil on this.

**Comment by Lomanic** [comment]
Fixed by #901
